src/cmd/cli/command/compose.go

```diff
@@ -539,26 +540,28 @@ func makeComposeLogsCmd() *cobra.Command {
 			}
 
 			tailOptions := cli.TailOptions{
-				Services: services,
 				Etag:     etag,
-				Since:    ts,
+				Filter:   filter,
+				LogType:  *logType,
 				Raw:      raw,
+				Services: services,
+				Since:    ts,
 				Verbose:  verbose,
-				LogType:  *logType,
 			}
 
 			return cli.Tail(cmd.Context(), provider, projectName, tailOptions)
 		},
 	}
 	logsCmd.Flags().StringP("name", "n", "", "name of the service")
 	logsCmd.Flags().String("etag", "", "deployment ID (ETag) of the service")
+	logsCmd.Flags().Bool("follow", false, "follow log output") // NOTE: -f is already used by --file
```

Comment:
I added this for compat w/ `docker logs --follow` (which is currently the default for us, but could change it the future)